src/store/reducer/zoomCourtSlice.ts

```diff
@@ -0,0 +1,34 @@
+import { MAX_ZOOM, MIN_ZOOM, ZOOM_STEP_SIZE } from "@/constants/zoomLimit";
+import { createSlice } from "@reduxjs/toolkit";
+
+interface ZoomState {
+  zoomScale: number;
+  minScale: number;
+  maxScale: number;
+}
+
+const initialState: ZoomState = {
+  zoomScale: 1,
+  minScale: MIN_ZOOM,
+  maxScale: MAX_ZOOM,
+};
+
+export const ZoomControlSlice = createSlice({
+  name: "ZoomControl",
+  initialState,
+  reducers: {
+    zoomIn: (state: ZoomState) => {
+      state.zoomScale = state.zoomScale * ZOOM_STEP_SIZE;
```

Comment:
Multiple by step or increase by step?

By using multiple, first time is 1 * 1.1 = 1.1
Then 1.1*1.1 = 1.21
Then 1.21*1.1 = 1.331?

or 1 +0.1 = 1.1
then ++0.1 = 1.2 then 1.3, 1.4 and so on.

Reply:
we changed the algorithm to add/subtract a fixed constance (0.1) in each step.
The min and max scale variable are removed from the slice.

Reply:
> And I don't see any Min and Max zoom implementation in this slice. If not used in this slice, they are just constants that are not needed in this slice, or you put them in this slice logic of zoom in and out.

we changed the algorithm to add/subtract a fixed constance (0.1) in each step.
The min and max scale variable are removed from the slice.